xt/42-reconciliation.pg

```diff
@@ -48,161 +47,272 @@ BEGIN;
 
     -- Run tests
 
---    PREPARE test AS SELECT count(*)
---                      FROM defaults
---                      WHERE setting_key = 'check_prefix';
---    SELECT results_eq('test',ARRAY[1],'check_prefix set');
-
---    UPDATE defaults
---    SET value = 'Recon gl test '
---    WHERE setting_key = 'check_prefix';
-
-    PREPARE test AS SELECT reconciliation__new_report_id(test_get_account_id('-11111'), 100, now()::date, false);
-    SELECT results_eq('test',ARRAY[1],'Create Recon Report');
+    \set report_end1 '''2001-02-03'''
+    \set report_end2 '''2001-03-04'''
+    
+    --TODO: Do we still need this?
+    PREPARE test AS SELECT count(*) = 1
```

Comment:
Maybe better to create issues for these TODOs and reference the issue number in the file?